doc/source/whatsnew/v0.17.0.rst

```diff
@@ -729,7 +729,7 @@ or it can return False if broadcasting can not be done:
 
    np.array([1, 2, 3]) == np.array([1, 2])
 
-Changes to Boolean Comparisons vs. None
+Changes to Boolean comparisons vs. none
```

Comment:
I'd probably leave `None` as in Python, and may be `boolean`, even if I see your point. Up to you, just a personal preference here.